### PR TITLE
SR-3: added homework 3

### PR DESCRIPTION
**Pull request by Sofiiar**
***Description***
- changed locators to alternative option
- optimised `test_saucedemo.py` to read locators from separate file 

***Screenshot***
Screenshot 2024-05-03 at 14 33 45

### CODE REVIEW

File: tests/sradu/homework_3/locators.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+from selenium.webdriver.common.by import By
+
+
+class LoginPageLocators:
+    TXT_USER_NAME = (By.CSS_SELECTOR, "input[name='user-name']")  # prev => By.ID, "user-name"
```
Comment (kudinov-fedor) [inline tests/sradu/homework_3/locators.py:5]: "#user-name"

File: tests/sradu/homework_3/locators.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    DIV_CART_ITEM = (By.XPATH, "//div[@data-test='inventory-item']")
+    # prev => By.ID, "checkout"
+    BTN_CHECKOUT = (By.XPATH, "//button[@id='checkout']")
+
+
+class CheckoutInfoLocators:
+    IP_FIRST_NAME = (By.CSS_SELECTOR, "input[data-test='firstName']")  # By.ID, "first-name"
+    IP_LAST_NAME = (By.CSS_SELECTOR, "input[data-test='lastName']")  # By.ID, "last-name"
+    IP_POSTAL_CODE = (By.CSS_SELECTOR, "input[data-test='postalCode']")  # By.ID, "postal-code"
+    BTN_CONTINUE = (By.CSS_SELECTOR, "input[data-test='continue']")  # By.ID, "continue"
```
Comment (kudinov-fedor) [inline tests/sradu/homework_3/locators.py:44]: and why you do not use ID here ?

**Review by kudinov-fedor — APPROVED** [review]

File: tests/sradu/homework_3/locators.py (hunk 1 above)
Comment (Sofiiar) [reply tests/sradu/homework_3/locators.py]: Fixed.

File: tests/sradu/homework_3/locators.py (hunk 2 above)
Comment (Sofiiar) [reply tests/sradu/homework_3/locators.py]: changed to # syntax

**Pull request merged**